contracts/tasks/runner/merge.ts

```diff
@@ -11,13 +11,13 @@ import { EContracts, type IMergeParams } from "../helpers/types";
 const DEFAULT_SR_QUEUE_OPS = 4;
 
 /**
- * Command to merge signup and message queues of a MACI contract
+ * Command to merge signups of a MACI contract
  */
-task("merge", "Merge signups and messages")
+task("merge", "Merge signups")
   .addParam("poll", "The poll id", undefined, types.string)
   .addOptionalParam("queueOps", "The number of queue operations to perform", DEFAULT_SR_QUEUE_OPS, types.int)
```

Comment:
this is not needed anymore

Reply:
should we delete whole file or only this task (it is only task in that file, but whatever you want)

Reply:
I meant the flag queueOps